FWCore/ParameterSet/interface/defaultModuleLabel.h

```diff
@@ -0,0 +1,30 @@
+#ifndef FWCore_ParameterSet_defaultModuleLabel_h
+#define FWCore_ParameterSet_defaultModuleLabel_h
+
+#include <string>
+#include <algorithm>
+
+namespace edm {
+  std::string defaultModuleLabel(std::string label) { // take by value because we'll copy it anyway
+    // remove all colons (module type may contain namespace)
+    label.erase(std::remove(label.begin(), label.end(), ':'), label.end());
+
+    // if the label is all uppercase, change it to all lowercase
+    // if the label starts with more than one uppercase letter, change n-1 to lowercase
+    // otherwise, change the first letter to lowercase
```

Comment:
I know that the logic is trivial, still I would have appreciated something along "taken from HLTrigger/HLTcore/interface/defaultModuleLabel.h" in a C++ comment or in the commit message :-(

Reply:
Good point, I added a comment noting the origin.

Reply:
Thanks :-)